runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -630,23 +630,23 @@ class WorkItemListPage {
   }
 
   clickExpandCurrentIterationIcon () {
-    return this.expandCurrentIterationIcon.click(); 
+    return this.expandCurrentIterationIcon.click();
   }
 
   get expandFutureIterationIcon () {
     return element(by.xpath (".//text()[contains(.,'Future Iterations')]/.."));
   }
 
   clickExpandFutureIterationIcon () {
-    return this.expandFutureIterationIcon.click(); 
+    return this.expandFutureIterationIcon.click();
```

Comment:
Is this associated with non contextual detail view?

Reply:
This is not, but it can't be avoided since there are changes in the file, and by linter setup when you save the file this correction happens automatically. So you can grant this.